canonical-web-and-design/maas-ui
# feat(ui): split KVMSettings into composable components

## Done

- Split `KVMSettings` into just the subcomponent `KVMConfigurationCard`
- Built view components `LXDSingleSettings` and `VirshSettings` from the subcomponent
- Moved `AuthenticationCard` and `DangerZoneCard` to `views/LXDSingleDetails/LXDSingleSettings` since they are only used there
- Made form cards no longer highlighted

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the "Settings" tab of a LXD single host and check that it looks the same apart from the cards are no longer highlighted
- Go to the "Settings" tab of a virsh host and check the same

## Fixes

Fixes canonical-web-and-design/app-squad#397
